src/components/List.js

```diff
@@ -1,19 +1,30 @@
-export default [
+const books = [
   {
     title: 'The Hunger Games',
     author: 'Suzanne Collins',
     id: 1,
+    category: 'Classic',
+    completed: 60,
+    currentChapter: 8,
   },
 
   {
     title: 'Dune',
     author: 'Frank Herbert',
     id: 2,
+    category: 'Fiction',
+    completed: 80,
+    currentChapter: 12,
   },
 
   {
     title: 'Capital in the Twenty-First Century',
     author: 'Suzanne Collins',
     id: 3,
+    category: 'Romance',
+    completed: 88,
+    currentChapter: 14,
   },
 ];
+
+export default books;
```

Comment:
- [ ] the books should be added to the initial state in Redux and you're supposed to access the store using "useselector" hook after that.